pkgs/applications/office/kitsas/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, mkDerivation, fetchFromGitHub, qmake, qtsvg, qtcreator, poppler, libzip, pkg-config }:
+
+mkDerivation rec {
+  pname = "kitsas";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "artoh";
+    repo = "kitupiikki";
+    rev = "v${version}";
+    sha256 = "1qac6cxkb45rs5pschsf2rvpa789g27shmrwpshwahqzhw42xvgl";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ qmake qtsvg poppler libzip ];
+
+  # We use a separate build-dir as otherwise ld seems to get confused between
+  # directory and executable name on buildPhase.
+  preConfigure = ''
+    mkdir build-linux
+    cd build-linux
+  '';
+
+  qmakeFlags = [
+    "../kitsas/kitsas.pro" "-spec" "linux-g++" "CONFIG+=release"
+  ];
+
+  preFixup = ''
+    make clean
+    rm Makefile
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share
+    cp kitsas $out/bin
+    cp $src/kitsas.png $out/share
+    cp $src/kitsas.desktop $out/share
```

Comment:
```suggestion
    cp $src/kitsas.desktop $out/share/applications
```

Reply:
Thanks, I made this change and tried two formatters on this `default.nix` file and included the changes (the `qmakeFlags` on separate lines). Further, I found a typo on the description (the associations word).

To be sure that there is that `applications` directory, I added it into the `mkdir` command above.

I didn't realize at all that the `applications` directory was needed (the icons were "working" in my case when I tried this locally).